### PR TITLE
ref(astro): Improve Getting Started introduction

### PR DESCRIPTION
**Pull request by Lms24**
This PR refactors the  Astro "Getting Started" intro section. I tried the suggested "ramp" approach for docs. Figured as Astro is new and relatively small it's ideal to experiment with. Feel free to close this PR if you wanna tackle this more generally. Appreciate feedback either way. 

image

Motivation for this change: Other than the meeting last week, we had users writing in ([example](https://github.com/getsentry/sentry-javascript/issues/9777)) about Cloudflare and Edge support and it seemed like our previous compatibility note didn't state clearly enough that these runtimes are currently not supported. 

Note: I also wanted to add numbering to the steps but this requires more fundamental changes to the common "getting started" template and I want to avoid this for now. IMO we should get rid of this common template. It's full of special cases already and I don't think we benefit much from it anymore. Probably collides with the rewrite anyway...

### CODE REVIEW

File: src/platform-includes/getting-started-primer/javascript.astro.mdx [hunk 1]
```diff
@@ -7,8 +9,12 @@ Please report any issues you encounter in our [Github Repository](https://github
...
 
-## Compatibility
+## Prerequisites
 
-The minimum supported Astro version is `3.0.0`.
-This SDK currently only works on Node runtimes.
-Non-Node runtimes, like Vercel's Edge runtime or Cloudflare Pages, are currently not supported.
+Before we get started, make sure you have the following:
+
+- A Sentry account and a project.
```
Comment (lizokm) [inline src/platform-includes/getting-started-primer/javascript.astro.mdx:16]:
```suggestion
- A Sentry account.
```

File: src/platform-includes/getting-started-primer/javascript.astro.mdx [hunk 2]
```diff
@@ -7,8 +9,12 @@ Please report any issues you encounter in our [Github Repository](https://github
...
+  - Don't have an account yet? Head over to [sentry.io](https://sentry.io/signup/), then return to this page.
+- An Astro project using Astro `3.0.0` or newer.
```
Comment (lizokm) [inline src/platform-includes/getting-started-primer/javascript.astro.mdx:18]:
```suggestion
- An Astro project that uses Astro `3.0.0` or newer.
```

File: src/platform-includes/getting-started-primer/javascript.astro.mdx [hunk 3]
```diff
@@ -7,8 +9,12 @@ Please report any issues you encounter in our [Github Repository](https://github
...
+- This SDK currently only works on Node runtimes.
+  - Non-Node runtimes, like Vercel's Edge runtime or Cloudflare Pages, are currently not supported.
```
Comment (lizokm) [inline src/platform-includes/getting-started-primer/javascript.astro.mdx:20]:
```suggestion
- Node runtimes.
  - This SDK currently only works on Node runtimes. Non-Node runtimes, like Vercel's Edge runtime or Cloudflare Pages, are currently not supported.
```

**Review by lizokm — APPROVED** [review]
LOVE these updates!

**Review by shanamatthews — APPROVED** [review]
Just a small tweak to fit our tab name convention.

**Pull request merged**